Changes to README.md

aaronleaa opened the pull request:
- Added List of supported countries

aaronleaa committed:
Update README.md

aaronleaa committed:
Update README.md

- Added list of supported countries

MunchDev reviewed (changes requested):
Need to create JSON file and reference link to it from readme file.

aaronleaa committed:
Added countries.json, edited README

- Removed country list from README and moved it to countries.json, for convenience purposes

aaronleaa committed:
Merge branch 'dev-country' of https://github.com/MunchDev/EpidemicSimulator into dev-country

aaronleaa committed:
Update countries.json

Removed []
